cpp/include/cudf/search.hpp

```diff
@@ -129,6 +129,32 @@ std::unique_ptr<column> upper_bound(table_view const& t,
 bool contains(column_view const& col, scalar const& value,
               rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**---------------------------------------------------------------------------*
```

Comment:
Please don't add the `---------------------------------------------------------------------------*` 
Doxygen does not seem to like it. https://docs.rapids.ai/api/libcudf/nightly/

Reply:
I just copied this from other headers.  Is there a header example that is a preferred template?

Reply:
https://github.com/rapidsai/cudf/blob/d85a6fa9f50cce75450d6053e76f085704700903/cpp/include/cudf/copying.hpp#L32

Reply:
Our built doxygen docs are a mess, but I never thought the ---- was the cause...

Reply:
Thanks for the reference.  We should probably fix all of the headers for this :-)